src/mlio/csv_reader.cxx

```diff
@@ -289,26 +289,40 @@ csv_reader::init_parsers_and_schema()
     auto col_beg = tbb::make_zip_iterator(idx_beg, name_beg, type_beg);
     auto col_end = tbb::make_zip_iterator(idx_end, name_end, type_end);
 
+    std::unordered_map<std::string, int> name_count;
```

Comment:
Although technically not necessary for an STL container, I prefer to brace-initialize all local variables to be consistent; meaning this line should be:
```
std::unordered_map<std::string, std::size_t> name_count{};
```

Reply:
Please use `std::size_t` instead of `int` for sizes and indices.

Reply:
Changed to size_t.